src/ansys/geometry/core/math/bbox.py

```diff
@@ -49,6 +50,7 @@ class BoundingBox2D:
     """
 
     @check_input_types
+    @deprecated_method(alternative="BoundingBox")
```

Comment:
Let's fully remove it in version 0.11. Have a look at other ``deprecated_method`` calls to see how to specify the version in which it will be removed